src/main/java/org/opensearch/searchrelevance/stats/events/EventStatName.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.searchrelevance.stats.events;
+
+import java.util.Arrays;
+import java.util.Locale;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import org.opensearch.searchrelevance.stats.common.StatName;
+
+import lombok.Getter;
+
+/**
+ * Enum that contains all event stat names, paths, and types
+ */
+@Getter
+public enum EventStatName implements StatName {
+    IMPORT_JUDGMENT_SCORE_GENERATIONS("import_judgment_score_generations", "judgment", EventStatType.TIMESTAMPED_EVENT_COUNTER),
+    LLM_JUDGMENT_SCORE_GENERATIONS("llm_judgment_score_generations", "judgment", EventStatType.TIMESTAMPED_EVENT_COUNTER),
+    UBI_JUDGMENT_SCORE_GENERATIONS("ubi_judgment_score_generations", "judgment", EventStatType.TIMESTAMPED_EVENT_COUNTER),;
```

Comment:
We renamed judgment score to judgement rating in https://github.com/opensearch-project/search-relevance/pull/64. Please rename constants/labels accordingly @q-andy